packages/amplify-cli-core/src/errors/index.ts

```diff
@@ -15,3 +15,15 @@ export class AppNotFoundError extends Error {}
 export class AppAlreadyDeployedError extends Error {}
 export class SchemaDoesNotExistError extends Error {}
 export class AngularConfigNotFoundError extends Error {}
+export class NotInitializedError extends Error {
+  public constructor() {
+    super();
+    (this.name = 'NotInitializedError'),
```

Comment:
Is there a reason not to do `this.name = 'NotInitializedError';` here? Same comment on the following line.

Reply:
Prettier was auto-formatting it, fixed it now :sweat_smile: